src/conduit/migrations/versions/72a996e5a6ca_tags.py

```diff
@@ -26,7 +26,7 @@ def upgrade():  # noqa: D103
             server_default=sa.text("gen_random_uuid()"),
             nullable=False,
         ),
-        sa.Column("name", sa.Unicode(length=50), nullable=False),
```

Comment:
Nowhere in the RealWorld.io spec does it say that name is limited to 50 chars. This is here probably due to my copy/pasting code from other projects.